### PR TITLE
Update Module.bsl

### PR DESCRIPTION
**Pull request by nixel2007**
Don't merge

### CODE REVIEW

File: src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 &НаКлиенте
 Процедура СоздатьКаталогЦели()
-А=0;
+Б=0;
```
Comment (nixel2007) [inline src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl]: MINOR Add spacing around this punctuator. [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3APunctuatorsIndent)

**Comment by nixel2007** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [Module.bsl#L19](https://github.com/silverbulleters/vanessa-agiler/blob/a24f7cc38b01c6d3d11f9ef930d36ab242e693f3/src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl#L19): Document this public procedure. [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3AUndocumentedPublicApi)
1. MINOR [Module.bsl#L85](https://github.com/silverbulleters/vanessa-agiler/blob/a24f7cc38b01c6d3d11f9ef930d36ab242e693f3/src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl#L85): Split this 121 characters long line (which is greater than 120 authorized). [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3ALineLength)

File: src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl (hunk 1 above)
Comment (silverbulleters-ci-admin) [inline src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl]: MINOR Add spacing around this punctuator. [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3APunctuatorsIndent)

**Comment by silverbulleters-ci-admin** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [Module.bsl#L19](https://github.com/silverbulleters/vanessa-agiler/blob/4cbd1f1c4efd58e5b342a83caf2777dd0c34088b/src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl#L19): Document this public procedure. [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3AUndocumentedPublicApi)
1. MINOR [Module.bsl#L85](https://github.com/silverbulleters/vanessa-agiler/blob/4cbd1f1c4efd58e5b342a83caf2777dd0c34088b/src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl#L85): Split this 121 characters long line (which is greater than 120 authorized). [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3ALineLength)

File: src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl (hunk 1 above)
Comment (silverbulleters-ci-admin) [inline src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl]: MINOR Add spacing around this punctuator. [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3APunctuatorsIndent)

File: src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 &НаКлиенте
 Процедура СоздатьКаталогЦели()
+Б=0;
```
Comment (silverbulleters-ci-admin) [inline src/cf/Catalogs/БизнесЦелиПроекта/Forms/ФормаЭлемента/Ext/Form/Module.bsl]: MINOR Add spacing around this punctuator. [rule](http://sonar.silverbulleters.org/coding_rules#rule_key=bsl%3APunctuatorsIndent)

**Pull request closed without merging**